package.json

```diff
@@ -15,6 +15,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "jshashes": "^1.0.8",
+    "lodash": "^4.17.20",
```

Comment:
Adding this because I didn't want to reimplement stuff like `sortBy` and `uniq`.